nethserver-ndpi.spec

```
@@ -3,13 +3,16 @@ Version: 1.2.0
 Release: 1%{?dist}
 Summary: Conifigure ndpi kernel modules
 Source: %{name}-%{version}.tar.gz
-BuildArch: noarch
+Source1: https://github.com/filippocarletti/xt_ndpi-kmod/releases/download/nDPI-2.8_3.10.0-957.el7/xt_ndpi.ko
```

Please change to NethServer URL as soon as NethServer/xt_ndpi-kmod#10 has been merged.

```suggestion
Source1: https://github.com/NehServer/xt_ndpi-kmod/releases/download/nDPI-2.8_3.10.0-957.el7/xt_ndpi.ko
```